healthcare/api-client/fhir/fhir_resources_test.py

```diff
@@ -91,124 +116,287 @@ def test_CRUD_search_resource(test_dataset, test_fhir_store, capsys):
         dataset_id,
         fhir_store_id,
         resource_type,
-        resource_id,
+        test_patient
     )
 
-    fhir_resources.update_resource(
+    out, _ = capsys.readouterr()
+
+    print(out)
+
+    assert "Got Patient resource" in out
+
+
+@pytest.mark.skip(reason='flaky test sometimes returns 403 errors, need to investigate')
```

Comment:
I'm cool with a skip as long as an issue gets opened up in the repo referencing this